===================================================================
RCS file: /srv/anoncvs.freedesktop.org/cvs/xorg/xserver/xorg/cfb/cfbrrop.h,v
rcsdiff: /srv/anoncvs.freedesktop.org/cvs/xorg/xserver/xorg/cfb/cfbrrop.h,v: warning: Unknown phrases like `commitid ...;' are present.
retrieving revision 1.4
retrieving revision 1.5
diff -u -r1.4 -r1.5
--- xorg.orig/xserver/xorg/cfb/cfbrrop.h	2005/07/03 07:01:15	1.4
+++ xorg/xserver/xorg/cfb/cfbrrop.h	2006/03/20 18:43:18	1.5
@@ -284,6 +284,7 @@
 #define RROP_UNROLL_CASE4(p,i)    RROP_UNROLL_CASE2(p,(i)+2) RROP_UNROLL_CASE2(p,i)
 #define RROP_UNROLL_CASE8(p,i)    RROP_UNROLL_CASE4(p,(i)+4) RROP_UNROLL_CASE4(p,i)
 #define RROP_UNROLL_CASE16(p,i)   RROP_UNROLL_CASE8(p,(i)+8) RROP_UNROLL_CASE8(p,i)
+#define RROP_UNROLL_CASE32(p,i)   RROP_UNROLL_CASE16(p,(i)+16) RROP_UNROLL_CASE16(p,i)
 #define RROP_UNROLL_CASE3(p)	RROP_UNROLL_CASE2(p,2) RROP_UNROLL_CASE1(p,1)
 #define RROP_UNROLL_CASE7(p)	RROP_UNROLL_CASE4(p,4) RROP_UNROLL_CASE3(p)
 #define RROP_UNROLL_CASE15(p)	RROP_UNROLL_CASE8(p,8) RROP_UNROLL_CASE7(p)
